locale/oc-lnc

```diff
@@ -0,0 +1,81 @@
+//! moment.js locale configuration
+//! locale : Occitan, lengadocian dialecte [oc-lnc]
+//! author : Quentin
+
+;(function (global, factory) {
+   typeof exports === 'object' && typeof module !== 'undefined'
+       && typeof require === 'function' ? factory(require('../moment')) :
+   typeof define === 'function' && define.amd ? define(['../moment'], factory) :
+   factory(global.moment)
+}(this, (function (moment) { 'use strict';
+
+
+var ocLnc = moment.defineLocale('oc-lnc', {
+    months : 'genièr_febrièr_març_abril_mai_junh_julhet_agost_setembre_octobre_novembre_decembre'.split('_'),
+    monthsShort : 'gen._febr._mar._abr._mai._jun._jul._ag._set._oct._nov._dec.'.split('_'),
+    monthsParseExact : true,
+    weekdays : 'dimenge_diluns_dimars_dimècres_dijòus_divendres_dissabte'.split('_'),
+    weekdaysShort : 'dg._dl._dr._dc._dj._dv._ds.'.split('_'),
+    weekdaysMin : 'Dg_Dl_Dt_Dc_Dj_Dv_Ds'.split('_'),
+    weekdaysParseExact : true,
+    longDateFormat : {
+        LT : 'H:mm',
+        LTS : 'H:mm:ss',
+        L : 'DD/MM/YYYY',
+        LL : 'D MMMM YYYY',
+        LLL : 'D MMMM YYYY H:mm',
+        LLLL : 'dddd D MMMM YYYY H:mm'
+    },
+    calendar : {
+        sameDay : function () {
+            return '[uèi a] LT';
+        },
+        nextDay : function () {
+            return '[deman a] LT';
+        },
+        nextWeek : function () {
+            return 'dddd [a] LT';
+        },
+        lastDay : function () {
+            return '[ièr a] LT';
+        },
+        lastWeek : function () {
+            return 'dddd [passat a] LT';
+        },
+        sameElse : 'L'
+    },
+    relativeTime : {
+        future : 'd\'aquí %s',
+        past : 'fa %s',
+        s : 'unas segondas',
+        m : 'una minuta',
+        mm : '%d minutas',
+        h : 'una ora',
+        hh : '%d oras',
+        d : 'un jorn',
+        dd : '%d jorns',
+        M : 'un mes',
+        MM : '%d meses',
+        y : 'un an',
+        yy : '%d ans'
+    },
+    ordinalParse: /\d{1,2}(r|n|t|è|a)/,
```

Comment:
Please change to `dayOfMonthOrdinalParse`. This was recently updated in master branch.